internal/operands/common-instancetypes/reconcile.go

```diff
@@ -321,6 +323,12 @@ func (c *CommonInstancetypes) reconcileVirtualMachineClusterInstancetypesFuncs()
 	for i := range c.virtualMachineClusterInstancetypes {
 		clusterInstancetype := &c.virtualMachineClusterInstancetypes[i]
 		funcs = append(funcs, func(request *common.Request) (common.ReconcileResult, error) {
+			if ignore, err := ignoreObjectOwnedByVirtOperator(request, clusterInstancetype.DeepCopy()); err != nil {
```

Comment:
Can you move this check outside of this closure? If `ignoreObjectOwnedByVirtOperator()` returns true, we don't need to add this closure to the `funcs` slice.

Reply:
A client is needed to retrieve the object from the API. I don't think this is possible outside the closure?

Reply:
You can pass the `*common.Request` also to the enclosing function. We do something similar with DataSources:
https://github.com/kubevirt/ssp-operator/blob/9759ddfe94f066df6cb0d97e727bf06342b6d7bd/internal/operands/data-sources/reconcile.go#L335